cdap-ui/cdap-ui-upgrade/index.js

```diff
@@ -140,7 +146,7 @@ var tasks = new Listr([
     exitOnError: true
   },
   {
-    title: 'Backing existing UI : <' + cdaphome + '>',
+    title: 'Backing existing UI : <' + (cdaphome ? cdaphome : 'unknown-path') + '>',
```

Comment:
The message was from @nitinmotgi. I'm not too strong on either of them. @nitinmotgi Thoughts?